src/main/java/edu/iis/mto/time/Order.java

```diff
@@ -28,13 +30,13 @@ public void submit() {
         requireState(State.CREATED);
 
         orderState = State.SUBMITTED;
-        subbmitionDate = LocalDateTime.now();
+        subbmitionDate = LocalDateTime.of(2022,3,31,0,0,0);
```

Comment:
to jest kod produkcyjny - dlaczego coś tu hadrkodujemy? To ma działać poprawnie "na produkcji" a nie tylko w teście!